ztest/ztest.go

```diff
@@ -232,17 +233,19 @@ func run(zq, ZQL, outputFormat string, inputs ...string) (string, error) {
 
 func loadInputs(inputs []string, zctx *resolver.Context) (zbuf.Reader, error) {
 	var readers []zbuf.Reader
-	for _, input := range inputs {
+	var names []string
+	for i, input := range inputs {
 		zr, err := detector.NewReader(detector.GzipReader(strings.NewReader(input)), zctx)
 		if err != nil {
 			return nil, err
 		}
 		readers = append(readers, zr)
+		names = append(names, strconv.Itoa(i))
```

Comment:
Instead of changing the `NewCombiner` api to accept this extra arg, I would suggest wrapping `zr` with something like this:
```golang
type namedReader struct {
    zbuf.Reader
    name string
}

func (r *namedReader) String() string {
    return r.name
}
```

Reply:
That sounds good

Reply:
Done